runhouse/rns/defaults.py

```diff
@@ -26,7 +26,7 @@ class Defaults:
         "default_autostop": -1,
         "use_spot": False,
         "use_local_configs": True,
-        "disable_data_collection": False,
+        "disable_data_collection": True,
```

Comment:
thoughts about leaving it on by default? otherwise we would not be getting the sentry error reports / cluster usage data

Reply:
Makes sense! I was using it for local Docker testing.